exec/java-exec/src/main/java/org/apache/drill/exec/store/sys/ExtendedOptionIterator.java

```diff
@@ -115,14 +143,20 @@ public boolean hasNext() {
   @Override
   public ExtendedOptionValueWrapper next() {
     final OptionValue value = mergedOptions.next();
-    final HashMap<OptionValue.Kind,String> typeMapping = new HashMap() {{
-      put(Kind.STRING,"VARCHAR");
-      put(Kind.DOUBLE,"FLOAT");
-      put(Kind.LONG,"BIGINT");
-      put(Kind.BOOLEAN,"BIT");
 
-    }};
-    return new ExtendedOptionValueWrapper(value.name, typeMapping.get(value.kind), value.accessibleScopes,value.getValue().toString(), value.scope);
+    final Status status;
+    if (value.accessibleScopes == AccessibleScopes.BOOT) {
+      status = Status.BOOT;
+    } else {
+      final OptionValue def = fragmentOptions.getDefault(value.name);
+      if (value.equalsIgnoreType(def)) {
```

Comment:
Use ternary operator.

Reply:
👍 Ok. Was avoiding refactoring the original code, just I copied as is from OptionsIterator. :)